addons/binding/org.openhab.binding.km200/src/main/java/org/openhab/binding/km200/internal/KM200Device.java

```diff
@@ -0,0 +1,385 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.km200.internal;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+import javax.xml.bind.DatatypeConverter;
+
+import org.apache.commons.lang.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParseException;
+import com.google.gson.JsonParser;
+
+/**
+ * The KM200Device representing the device with its all capabilities
+ *
+ * @author Markus Eckhardt - Initial contribution
+ *
```

Comment:
Please kill some whitespace here and on line 31

Reply:
Removed...